server/src/models-abstract/measurable-model.js

```diff
@@ -25,6 +29,26 @@ class MeasurableModel extends ModelPostgres {
     );
   }
 
+  /**
+   * @return {Promise<boolean>}
+   */
+  setJudgementPending() {
+    return this.model.update({
```

Comment:
I kind of hate to say this, but can we move this logic up out of the db layer and into the model layer or similar? So the model layer has a method for ``shouldupdateStateToJudgementPending``, and another for ``updateStateToJudgementPending``. That way we could later have hooks here to add notifications, and other things. For instance, the ``resulution.js`` bot should run here.